resources/web/wwi/Parser.js

```diff
@@ -443,7 +464,8 @@ export default class Parser {
     } else if (type === 'solid' || type === 'robot') {
       newNode = new WbSolid(id, translation, scale, rotation);
       if (type === 'robot') {
-        const window = (node.hasAttribute('window') && node.getAttribute('window') != "<generic>") ? node.getAttribute('window') : 'generic';
+        const window = (node.hasAttribute('window') && node.getAttribute('window') !== '<generic>')
+          ? node.getAttribute('window') : 'generic';
```

Comment:
Te best practice is to split the line after the operator
```suggestion
        const window = (node.hasAttribute('window') && node.getAttribute('window') !== '<generic>') ?
          node.getAttribute('window') : 'generic';
```

Reply:
The linter tells me that `?` should be at the beginning of the line

Reply:
So, then it's ok like.
It's just strange because this is the typical example I found when searching of javascript best practice.